src/publish.proj

```diff
@@ -23,5 +26,33 @@
   <Target Name="Build" DependsOnTargets="CreateContainerName">
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
-  </Target>  
-</Project>
+  </Target>
+  
+  <PropertyGroup>
+    <PackageDownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)AzureTransfer\$(ConfigurationGroup)</PackageDownloadDirectory>
+    <FinalPublishPattern>$(PackageDownloadDirectory)\**\*.nupkg</FinalPublishPattern>
+    <FinalSymbolsPackagesPattern>$(PackageDownloadDirectory)\**\*.symbols.nupkg</FinalSymbolsPackagesPattern>
+  </PropertyGroup>
+  
+  <Target Name="PublishToAzureBlobFeed" >
+    <ItemGroup>
+      <ItemsToPush Include="$(FinalPublishPattern)" Exclude="$(FinalSymbolsPackagesPattern)" />
+    </ItemGroup>
+    <Error Condition="'@(ItemsToPush)'==''" Text="ItemsToPush for packages is empty." />
+    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
+                    AccountKey="$(AccountKey)"
+                    ItemsToPush="@(ItemsToPush)"
+                    Overwrite="$(PublishOverwrite)" />
+  </Target>
+  
+  <Target Name="PublishSymbolsToAzureBlobFeed" Condition="'$(PublishSymbols)' == 'true'" >
+    <ItemGroup>
+      <ItemsToPush Include="$(FinalSymbolsPackagesPattern)" />
+    </ItemGroup>
+    <Error Condition="'@(ItemsToPush)'==''" Text="ItemsToPush for symbols is empty." />
+    <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
+                    AccountKey="$(AccountKey)"
```

Comment:
For symbol packages I would expect they need to be published into a flat directory and not go into the feed, otherwise I expect they will likely override the real package. I don't expect these to really be indexed like our normal packages but honestly I don't know our current plan for them. @dagood is our symbols expert and might be able to give us some thoughts.

Reply:
Did the `PublishFlatContainer="true"` fix this? I can't tell what was commented on here because of force pushing.

Yes, trying to publish the library package and symbol packages to the same feed would be a conflict, because the nuspecs (containing id/version) are identical.